### PR TITLE
Move AppealsController#tasks to TasksController#for_appeal

### PR DESCRIPTION
**Pull request by lowellrex**
Consolidates task-related actions in the TasksController. Refactor that seemed timely given the amount of related work to come from #6703.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -100,6 +100,15 @@ def certification_header(title)
   end
   helper_method :certification_header
 
+  # https://stackoverflow.com/a/748646
+  def no_cache
```
Comment (mdbenjam) [inline app/controllers/application_controller.rb]: I do not think we should make every endpoint not cache. I would say most endpoints should.

Comment (mdbenjam) [reply app/controllers/application_controller.rb]: Maybe this is a reasonable alternative: https://stackoverflow.com/questions/11966766/hitting-browser-back-button-shows-json-rather-than-html-using-rails-d3-js

Comment (mdbenjam) [reply app/controllers/application_controller.rb]: Oh JK for some reason I thought this was a before action applied to everything 😅 ignore what I said.

Comment (lowellrex) [reply app/controllers/application_controller.rb]: Definitely still in favor of looking into a different approach for the no-cache stuff... though maybe that'll be a later PR

**Pull request merged**